imagetool/src/main/java/com/oracle/weblogic/imagetool/util/Utils.java

```diff
@@ -237,6 +237,32 @@ private static void processError(Process process) throws IOException {
                     "docker command failed with error: " + stringBuilder.toString());
         }
     }
+    //
+    //    public static boolean isJavaInstalled(String dockerImage) throws IOException, InterruptedException {
+    //        boolean result = false;
+    //        List<String> dockerInspectCommand = new ArrayList<>();
+    //        dockerInspectCommand.add("docker");
+    //        dockerInspectCommand.add("inspect");
+    //        dockerInspectCommand.add("--format='{{range .Config.Env}}{{println .}}{{end}}'");
+    //        dockerInspectCommand.add(dockerImage);
+    //        ProcessBuilder processBuilder = new ProcessBuilder(dockerInspectCommand);
+    //        Process process = processBuilder.start();
+    //        BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
+    //        String line;
+    //        while ((line = reader.readLine()) != null) {
+    //            System.out.println(line);
+    //            if (line.trim().startsWith("JAVA_HOME")) {
+    //                result = true;
+    //            }
+    //        }
+    //
+    //        int exitVal = process.waitFor();
+    //        if (exitVal != 0) {
+    //            throw new IOException("ERROR:  Unable to inspect image " + dockerImage + ",
+    //            docker returned " + exitVal);
+    //        }
+    //        return result;
+    //    }
```

Comment:
Do we need to check for java version and short circuit here if it is lower than the required version ?

Reply:
Maybe, but how do you know what Java version the supplied Middleware installer requires?  Doesn't this need to be an exercise for the user?

Reply:
We can either let the installer fail later if the jdk in the image is not supported such as openjdk or not supported version.  Otherwise, we have to build a map for the installer and jdk requirements,  We can probably leave it to the user and I think if the installer checks it then it should fail very soon.